iothub/service/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -125,7 +52,7 @@ public static async Task<ErrorCode> GetExceptionCodeAsync(HttpResponseMessage re
                     if (messageFields != null
                         && messageFields.TryGetValue(MessageFieldErrorCode, out string errorCodeObj))
                     {
-                        // The result of TryParse is not being tracked since errorCodeValue has already been initialized to a default value of InvalidErrorCode.
+                        // The result of TryParse is not being tracked since errorCodeValue has already been initialized to a default value of Unknown.
                         _ = int.TryParse(errorCodeObj, NumberStyles.Any, CultureInfo.InvariantCulture, out errorCodeValue);
```

Comment:
It doesn't matter if it was initialized before this call because any `out` parameter must be set by the callee.